File: PN3.cpp
// Template for probplems on Numbers

#include<iostream>
using namespace std;

class Number
{
    private:
        int iNo;        // Charactristics

    public:
        // Behaviours
        void Accept()       // Setter
        {
            cout<<"Enter the value : "<<endl;
            cin>>this->iNo;
        }
		
		void Display()
		{
           int iCnt = 0;
		   for( iCnt = 1;  iCnt<=iNo; iCnt++)
           {
              cout<<iCnt<<endl;      //  4
           }
		}   

       
};

int main()
{
    Number nobj;

    nobj.Accept();
    nobj.Display();

 

    return 0;
}

File: PN3.java
import java.lang.*;
import java.util.*;

class Number
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter number : ");
		this.iNo = sobj.nextInt();
	}
	
	public void Display()
	{
		int iCnt = 0;
		
		if(iNo < 0)
		{
			System.out.println("Plese enter positive number");
			return;
		}
		
		for(iCnt = 0; iCnt<iNo;iCnt++)
		{
			System.out.println("Hello");
		}
	}
}

class PN24
{
	public static void main(String a[])
	{
		Number nobj = new Number();
		
		nobj.Accept();
		nobj.Display();
		
	
	}
}